bazel/repository_locations.bzl

```diff
@@ -1503,4 +1503,5 @@ def _compiled_protoc_deps(locations, versions):
             license_url = "https://github.com/protocolbuffers/protobuf/blob/v{version}/LICENSE",
         )
 
+REPOSITORY_LOCATIONS_SPEC["com_github_intel_ipp_crypto_crypto_mb_fips"] = REPOSITORY_LOCATIONS_SPEC["com_github_intel_ipp_crypto_crypto_mb"]
```

Comment:
this breaks our dependency tooling

Reply:
i think if you do something like this:

```diff
diff --git a/api/bazel/envoy_http_archive.bzl b/api/bazel/envoy_http_archive.bzl
index 15fd65b2af..37e40df733 100644
--- a/api/bazel/envoy_http_archive.bzl
+++ b/api/bazel/envoy_http_archive.bzl
@@ -1,6 +1,6 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-def envoy_http_archive(name, locations, **kwargs):
+def envoy_http_archive(name, locations, location=None, **kwargs):
     # `existing_rule_keys` contains the names of repositories that have already
     # been defined in the Bazel workspace. By skipping repos with existing keys,
     # users can override dependency versions by using standard Bazel repository
@@ -10,7 +10,7 @@ def envoy_http_archive(name, locations, **kwargs):
         # This repository has already been defined, probably because the user
         # wants to override the version. Do nothing.
         return
-    location = locations[name]
+    location = locations[location or name]
 
     # HTTP tarball at a given URL. Add a BUILD file if requested.
     http_archive(
```

you should be able to reuse the existing repo location for the new definition - i didnt test it but i think this shoudl work in theory

Reply:
(probs change the nme of the existing `location` var to prevent confusion/clash)

Reply:
Thanks, I submit a new commit to fix this.